distribution/docs/src/main/resources/_contents/_queries/filter-intro-contents.adoc

```diff
@@ -0,0 +1,94 @@
+
+=== Filters
+
+An OGC Filter is a http://www.opengeospatial.org/standards/filter[Open Geospatial Consortium (OGC) standard] that describes a query expression in terms of Extensible Markup Language (XML) and key-value pairs (KVP). 
+The ${ddf-catalog} Framework does not use the XML representation of the OGC Filter standard. ${branding} instead uses the Java implementation provided by http://geotools.org/[Geotools].
+Geotools provides Java equivalent classes for OGC Filter XML elements.
+Geotools originally provided the standard Java classes for the OGC Filter Encoding 1.0 under the package name `org.opengis.filter`.
+The same package name is used today and is currently used by ${branding}. 
+Java developers do not parse or view the XML representation of a Filter in ${branding}. Instead, developers use only the Java objects to complete query tasks.
+
+Note that the `ddf.catalog.operation.Query` interface extends the `org.opengis.filter.Filter` interface, which means that a Query object is an OGC Java Filter with Query Options.
+
+.A Query is an OGC Filter
+[source,java]
+----
+public interface Query extends Filter
+----
+
+==== `FilterBuilder` API
+
+To abstract developers from the complexities of working with the Filter interface directly and implementing the ${branding} Profile of the Filter specification, the ${ddf-catalog} includes an API, primarily in `${ddf-branding}.filter`, to build Filters using a fluent API.
+
+To use the `FilterBuilder` API, an instance of `${ddf-branding}.filter.FilterBuilder` should be used via the OSGi registry.
+Typically, this will be injected via a dependency injection framework.
+Once an instance of `FilterBuilder` is available, methods can be called to create and combine Filters.
+
+[TIP]
+====
+The fluent API is best accessed using an IDE that supports code-completion.
+For additional details, refer to the [Catalog API Javadoc].
+====
+
+===== Boolean Operators
+
+Filters use a number of boolean operators.
+
+`FilterBuilder.allOf(Filter ...)`:: creates a new Filter that requires all provided Filters are satisfied (Boolean AND), either from a List or Array of Filter instances.
+
+`FilterBuilder.anyOf(Filter ...)`:: creates a new Filter that requires all provided Filters are satisfied (Boolean OR), either from a List or Array of Filter instances.
+
+`FilterBuilder.not(Filter filter)`:: creates a new Filter that requires the provided Filter must not be match (Boolean NOT).
```

Comment:
fixed. Went with `must not match` unless we think `be matched` is more accurate. I think they're equivalent & the simpler version is better, but I can be persuaded.